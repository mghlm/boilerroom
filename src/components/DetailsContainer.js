import React, { Component } from 'react';
import { View } from 'react-native';

class DetailsContainer extends Component {
  render() {
    return (
      <View style={styles.DetailsContainer}>
        {this.props.children}
      </View>
    );
  }
}

const styles = {
  DetailsContainer: {
    flex: 6,
    flexDirection: "column",
    justifyContent: "flex-start",
    alignItems: "stretch",
    marginTop: -10,
    marginBottom: 15
  }
};

export default DetailsContainer;
