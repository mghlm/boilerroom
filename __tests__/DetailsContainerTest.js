import 'react-native'
import React from 'react';
import DetailsContainer from '../src/components/DetailsContainer';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<DetailsContainer />).toJSON();
expect(tree).toMatchSnapshot();
});
