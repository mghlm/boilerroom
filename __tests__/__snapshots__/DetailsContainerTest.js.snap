// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "flex": 6,
      "flexDirection": "column",
      "justifyContent": "flex-start",
      "marginBottom": 15,
      "marginTop": -10,
    }
  }
/>
`;
